src/data/lingui/es/messages.json

```diff
@@ -354,69 +354,71 @@
+    "AssetsItemEdit.error.name": "Es necesario un nombre de activo válido",
+    "AssetsItemEdit.error.symbol": "Es necesario un símbolo de activo válido",
+    "AssetsItemEdit.toast": "Activo actualizado",
+    "More.action.about": "Información",
+    "More.action.support": "Asistencia",
+    "More.description.about": "Acerca del monedero Jibrel",
+    "More.title": "Más",
+    "common.DigitalAssetEditForm.address.label": "Dirección del activo",
+    "common.DigitalAssetEditForm.decimals.label": "Decimales",
+    "common.DigitalAssetEditForm.name.label": "Nombre",
+    "common.DigitalAssetEditForm.submit": "Guardar",
+    "common.DigitalAssetEditForm.symbol.label": "Símbolo",
+    "common.DigitalAssetEditForm.title": "Editar activo",
     "AssetsItemDetails.address.title": "Dirección del activo",
     "AssetsItemDetails.name.decimals": "Decimales",
     "AssetsItemDetails.name.symbol": "Símbolo",
     "AssetsItemDetails.name.title": "Nombre",
     "AssetsItemDetails.title": "Detalles del activo",
     "WalletPasswordForm.Forgot.action.back": "Volver a intentarlo",
     "WalletPasswordForm.Forgot.action.forgot": "¿Se ha olvidado?",
-    "WalletPasswordForm.Forgot.description": "",
+    "WalletPasswordForm.Forgot.description": "Lo sentimos, no se puede recuperar la contraseña de seguridad. Si desea restablecer Jwallet a su estado inicial y dejar su contraseña de seguridad antigua, póngase en contacto con el servicio de asistencia técnica.",
     "WalletPasswordForm.Forgot.title": "Contraseña de seguridad olvidada",
     "HistoryList.Empty.text": "Parece que aún no ha efectuado ninguna transacción.",
-    "Send.StepOneForm.global.error.text": "",
-    "Send.StepOneForm.global.error.title": "",
-    "Send.StepOneForm.node.error.title": "",
-    "HistoryItemCancel.agreement.header": "",
+    "Send.StepOneForm.global.error.text": "El importe introducido + tarifa excede el saldo disponible. Corrija la cantidad o el valor del precio del gas, por favor.",
+    "Send.StepOneForm.global.error.title": "Fondos insuficientes",
+    "Send.StepOneForm.node.error.title": "Error de cadena de bloques",
+    "HistoryItemCancel.agreement.header": "Introduzca la contraseña de seguridad",
     "Send.title": "Enviar",
-    "Send.StepOneForm.toast.network": "",
-    "Wallets.title": "",
-    "WalletsImport.errors.privateKeyTooLong": "",
-    "WalletsImport.errors.xprvTooLong": "",
-    "WalletsImport.errors.xpubTooLong": ""
+    "Send.StepOneForm.toast.network": "Error de red. No puede solicitar gasPrecio",
+    "Wallets.title": "Mis monederos",
+    "WalletsImport.errors.privateKeyTooLong": "La clave privada de Ethereum no puede tener más de 64 caracteres",
+    "WalletsImport.errors.xprvTooLong": "BIP32 XPRV no puede tener más de { length } caracteres",
+    "WalletsImport.errors.xpubTooLong": "BIP32 XPUB no puede tener más de { length } caracteres",
+    "HistoryList.Empty.text.filtered": "",
+    "WalletsMigration.description": ""
```

Comment:
We didn't upload these ids to Lokalise, or translators forgot to translate them?

Reply:
Seems, that we didn't upload them